src/test/java/io/pravega/connectors/flink/utils/SetupUtils.java

```diff
@@ -59,22 +59,27 @@
     private static final String PASSWD_FILE = "passwd";
     private static final String KEY_FILE = "key.pem";
     private static final String CERT_FILE = "cert.pem";
-    private static final String BOOKIE_KEYSTORE_FILE = "bookie.keystore.jks";
-    private static final String BOOKIE_TRUSTSTORE_FILE = "bookie.truststore.jks";
-    private static final String BOOKIE_KEYSTORE_PASSWD_FILE = "bookie.keystore.jks.passwd";
+    private static final String STANDALONE_KEYSTORE_FILE = "standalone.keystore.jks";
+    private static final String STANDALONE_TRUSTSTORE_FILE = "standalone.truststore.jks";
+    private static final String STANDALONE_KEYSTORE_PASSWD_FILE = "standalone.keystore.jks.passwd";
 
     private final PravegaGateway gateway;
 
     // Manage the state of the class.
     private final AtomicBoolean started = new AtomicBoolean(false);
 
-    // Set to true to enable Pravega authentication
+    // auth enabled by default. Set it to false to disable Pravega authentication
```

Comment:
"auth" represents both authentication and authorization. So, we could say `... to disable Pravega authentication and authorization` instead.

Reply:
Fixed the comment